lib/subcommands/build_release_notes.py

```diff
@@ -42,45 +36,62 @@
 RELEASE_FILE_LAYOUT = "release"
 
 
-def write_version_info(release_tag, file_path, versions_repo, packages):
+def write_version_info(file_path, release_date, build_info):
     """
     Write release information to a file.
     It contains packages names, branches and commit IDs.
+
+    Args:
+        file_path (str): path to the output release file
+        release_date (str): date of the release
+        build_info (dict): information about the build
     """
-    LOG.info("Creating release {release_tag} information".format(**locals()))
+
+    release_tag = "%s-%s" % (build_info['version'], release_date)
+
+    desired_keys = ['sources', 'version', 'release']
+    packages = [dict({k: v for k,v in pkg_info.items() if k in desired_keys},
+                     name=pkg_name)
+                for pkg_name, pkg_info in build_info['packages'].items()]
 
     release_file_info = {
         "title": RELEASE_FILE_TITLE,
         "layout": RELEASE_FILE_LAYOUT,
         "release_tag": release_tag,
-        "builds_commit": str(repository.GitRepository(".").head.commit.hexsha),
-        "versions_commit": str(versions_repo.head.commit.hexsha),
+        "packages": packages,
+        "builds_commit": build_info['builds_repo_commit_id'],
+        "versions_commit": build_info['versions_repo_commit_id'],
     }
 
-    packages_info = build_info.query_pkgs_info(
-        packages, ["name", "version", "release", "sources"], True)
-
-    release_file_info["packages"] = [packages_info[k]
-                                     for k in sorted(packages_info.keys())]
-
     LOG.info("Writing release {release_tag} information to file: {file_path}"
              .format(**locals()))
     with open(file_path, "w") as version_info_file:
         release_file_content = RELEASE_FILE_CONTENT_TEMPLATE.format(
-            header_yaml=yaml.dump(release_file_info, default_flow_style=False))
+            header_yaml=yaml.safe_dump(release_file_info, default_flow_style=False))
         version_info_file.write(release_file_content)
 
 
-def run(CONF):
-    versions_repo = setup_versions_repository(CONF)
+def read_info_files(info_files_path):
+
+    info = {}
+    for file_name in os.listdir(info_files_path):
+        if not file_name.endswith('.json'):
+            continue
 
-    version_milestone = read_version_and_milestone(versions_repo)
+        with open(os.path.join(info_files_path, file_name)) as json_file:
+            file_content = json.load(json_file)
+
+        node_name = file_name.replace('.json','')
+        if node_name == 'build':
+            # leave build information at the top-level
+            info.update(file_content)
```

Comment:
I don't see any advantage on having a special treatment to leave this at the top level.